packages/core/src/hydration/event_replay.ts

```diff
@@ -79,3 +118,78 @@ export function setJSActionAttribute(
     }
   }
 }
+
+function setEventReplayer(dispatcher: Dispatcher) {
+  dispatcher.setEventReplayer(queue => {
+    for (const event of queue) {
+      handleEvent(event);
+    }
+  });
+}
+
+/**
+ * TODO: Update documentation here.
+ *
+ * @param target Component, Directive or DOM Node.
+ */
+export function getLViewByElement(target: any): LView|null {
```

Comment:
Let's specify a more precise type here and do type casting (if needed) at call site.
```suggestion
export function getLViewByElement(target: HTMLElement): LView|null {
```

Reply:
done